bin/oref0-pump-loop.sh

```diff
@@ -733,7 +733,7 @@ function refresh_profile {
     else
         profileage=$1
     fi
-    find settings/ -mmin -$profileage -size +5c | grep -q settings.json && echo -n "Settings less than $profileage minutes old. " \
+    file_is_recent_and_min_size settings/settings.json $profileage | grep -q settings.json && echo -n "Settings less than $profileage minutes old. " \
```

Comment:
This line was incorrect - there's a stray "|grep -q settings.json" left over, which causes it to think the settings always need refreshing regardless of their actual age. Fixed in later commit.